web/versions.go

```diff
@@ -282,10 +351,18 @@ func getLatestVersion(c echo.Context) error {
 	if err != nil {
 		return err
 	}
-	version, err := registry.FindLatestVersion(getSpace(c), appSlug, ch)
+	space := getSpace(c)
+	version, err := registry.FindLatestVersion(space, appSlug, ch)
 	if err != nil {
 		return err
 	}
+	oVersion, e := override(c, version)
+	if e != nil {
+		return e
+	}
+	if oVersion != nil {
+		version = oVersion
+	}
```

Comment:
Idem, we can avoid those 3 lines by making `override` return the original version if there is no overriding